docs/reference/extensions/django-framework.rst

```diff
@@ -60,14 +61,10 @@ updates the Pebble plan to use the asynchronous workers. If you have ``gevent``
 installed in your rock but decided to use ``sync`` workers instead you can do
 so by using the ``--args`` parameter of ``docker run``:
 
-.. code-block:: shell
-  :caption: Use sync workers instead of gevent
-   $ docker run \
-       --name django-container \
-       -d -p 8138:8000 \
-       django-image:1.0 \
-       --args django sync
+.. code-block:: bash
 
+   docker run --name django-container -d -p 8138:8000 django-image:1.0 \
+   --args django sync
```

Comment:
What is the reason for picking port 8138 please?

Reply:
It is just a random port. Port 8000 in the host might be unavailable.

Reply:
Let's do 8000, if it isn't available the user can change it